### PR TITLE
[14.0] [IMP] - Generate INE (National Institute of Statistics of Spain) file.

### PR DESCRIPTION
**Pull request by miguelpadin**
Generates a file with the information required by the National Institute of Statistics of Spain (origin of guests, occupation, prices, ...)

https://user-images.githubusercontent.com/6873385/124429569-565cde80-dd6e-11eb-8751-f087e0de714c.mp4

### CODE REVIEW

**Comment by miguelpadin** [comment]
https://user-images.githubusercontent.com/6873385/124429569-565cde80-dd6e-11eb-8751-f087e0de714c.mp4

File: pms_l10n_es/wizards/wizard_ine.py [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+        )
+        revpar = round(
+            sum_group_price[0]["price"] / (available_rooms * last_day.day), 2
+        )
+        self.revpar = revpar
+        return revpar
+
+    @api.model
+    def ine_calculate_capacity(self):
+        # TODO: Review if calculate like below or get the info from property.ine_seats
```
Comment (miguelpadin) [inline pms_l10n_es/wizards/wizard_ine.py:388]: @DarioLodeiros @OsoTranquilo

**Comment by miguelpadin** [comment]
@DarioLodeiros

**Review by DarioLodeiros — APPROVED** [review]

**Pull request merged**